### PR TITLE
made the placeholder of the tooltip fillable from config

### PR DESCRIPTION
**Pull request by abdelhamiderrahmouni**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ export default class Tooltip {
     this.backgroundColor = config.backgroundColor;
     this.textColor = config.textColor;
     this.editorId = config.holder ? config.holder : 'editorjs';
+    this.inputPlaceholder = config.placeholder ? config.placeholder : "Add a tooltip";
```
Comment (miguelnietoa) [inline src/index.js:46]: @abdelhamiderrahmouni I have some suggestions to follow style guides used in the project:

- We could take advantage of the nullish coalescing operator (`??`)
- Consider using single quotes for the default value
- How about renaming `this.inputPlaceholder` to `this.placeholder` for consistency?

```suggestion
    this.placeholder = config.placeholder ?? 'Add a tooltip';
```

Would you mind applying the nullish coalescing operator on lines 42 and 45 as well? 🙌🏻

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
@abdelhamiderrahmouni thanks for this great contribution! Allowing to customize the input's placeholder is very useful! 👏🏻 

Could you please make sure to include the `placeholder` parameter in both the **Usage** and **Config Params** sections of the README file?

**Comment by abdelhamiderrahmouni** [comment]
> @abdelhamiderrahmouni thanks for this great contribution! Allowing to customize the input's placeholder is very useful! 👏🏻
> 
> Could you please make sure to include the `placeholder` parameter in both the **Usage** and **Config Params** sections of the README file?
Sure i would be happy to.

**Review by miguelnietoa — APPROVED** [review]

**Review by MarioRodriguezS — APPROVED** [review]

**Comment by miguelnietoa** [comment]
Hey @abdelhamiderrahmouni, I moved forward with the suggested changes in order to release this feature as soon as possible 🚀

**Comment by miguelnietoa** [comment]
@abdelhamiderrahmouni We just released [`v1.2.0`](https://github.com/kommitters/editorjs-tooltip/releases/tag/v1.2.0) which includes this new `placeholder` option. Thanks for contributing! 💯

**Pull request merged**